### ISSUE TITLE
Loss when training the text encoder is zero

### ISSUE THREAD

**Issue by IceIce1ce** [Lvchangze/SpikeCLIP#1]
Hi author, thanks for your great code. I found that when I run your training script on the CIFAR10 dataset for training the text encoder, the loss is always zero. This make the accuracy of the text encoder branch is not good. I do not know why this situation happens although I installed the default environment following your requirements.txt. Can you explain this?
